Makefile

```diff
@@ -91,7 +91,7 @@ vet:
 # Generate code
 .PHONY: generate
 generate: $(CONTROLLER_GEN)
-	$(CONTROLLER_GEN) object:headerFile=./hack/boilerplate.go.txt paths=./api/...
+	$(CONTROLLER_GEN) object:headerFile=./hack/boilerplate/boilerplate.go.txt paths=./api/...
```

Comment:
this should use `boilerplate.generatego.txt` since generated files should not have the year in the copyright